canvas_modules/common-canvas/__tests__/common-properties/properties-controller-test.js

```diff
@@ -2407,3 +2408,22 @@ describe("Properties Controller getTopLevelActiveGroupId", () => {
 
 	});
 });
+
+describe("Properties Controller setTopLevelActiveGroup", () => {
+	it("should set the top level active group id in properties controller", () => {
+		reset();
+		const renderedObject = testUtils.flyoutEditorForm(tabParamDef);
+		controller = renderedObject.controller;
+		const wrapper = renderedObject.wrapper;
+
+		controller.setTopLevelActiveGroupId("Primary3");
+		let topLevelActiveGroupId = controller.getTopLevelActiveGroupId();
+		expect(topLevelActiveGroupId).to.equal("Primary3");
```

Comment:
Are we able to validate on the UI side that this actually changes the active tab?